merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/LocalMissionModelService.java

```diff
@@ -171,9 +187,9 @@ public Map<String, SerializedValue> getModelEffectiveArguments(final String miss
          MissionModelLoadException,
          InvalidArgumentsException
   {
-    return this.loadConfiguredMissionModel(missionModelId)
-               .getEffectiveArguments(arguments);
```

Comment:
`MissionModelFacade#getEffectiveArguments` is no longer used anywhere. Can we remove it?

Reply:
Removed